frontend/src/utils/dateUtil.ts

```diff
@@ -3,7 +3,12 @@ export function formatDate(dateStr: string | null) {
   if (dateStr === "" || dateStr === null) {
     return "";
   }
-  const date = new Date(dateStr);
+
+  const [year, month, day] = dateStr.split("-").map(Number);
+
+  // Create a new Date object using the local time
+  const date = new Date(year, month - 1, day);
+
   const options: Intl.DateTimeFormatOptions = {
     year: "numeric",
     month: "long",
```

Comment:
this is the format date function we call from `SearchResultListItem` - it needs to be updated to be called from the local time instead of just the UTC time (which is I think what happens when you just do `new Date(dateStr)`

Reply:
This subtracts 1 from the month?

Reply:
The month is just 0-based - you can open chrome and enter 3 for the month - you'll see it's April:

![image](https://github.com/navapbc/simpler-grants-gov/assets/93001277/e599249a-2330-49d6-bdd4-40bc1a4470c8)

Reply:
@andycochran I think it's ready for review now